src/UI/SignTransactionsModals/SignWithDeviceModal/components/components/ConfirmReceiver/ConfirmReceiver.tsx

```diff
@@ -110,16 +120,20 @@ const ConfirmReceiverComponent = ({
             </span>
           )}
 
-          <CopyButton
-            text={receiver}
-            className={styles?.receiverCopy}
-            copyIcon={customCopyIcon}
-          />
+          {shouldTrimReceiver && (
```

Comment:
Why show copybutton only when receiver should be trimmed? What does that have to do with this?

Reply:
Restored. 👍